static/src/javascripts/projects/commercial/modules/dfp/Advert.js

```diff
@@ -134,6 +137,34 @@ class Advert {
         }
         updateAdvertMetric(this, 'stopRendering', getCurrentTime());
     }
+
+    static filterClasses(
+        oldClasses: Array<string>,
+        newClasses: Array<string>
+    ): Array<string> {
+        const classesToRemove: Array<string> = [];
+        for (let i = 0; i < oldClasses.length; i += 1) {
+            const currentClass: string = oldClasses[i];
+            if (!newClasses.includes(currentClass)) {
+                classesToRemove.push(currentClass);
+            }
+        }
```

Comment:
Can this be done with a filter?

Reply:
Thanks! Stayed away with it as wasn't sure if polyfilled or not and initial implementation was more elaborate.